CHANGES.md

```diff
@@ -121,7 +122,7 @@ Stay tuned, as this is only the first part of the overhaul. We’ll be adding mo
 
 ### Documentation
 
-- The [Asset Versioning and Caching Guide]([https://docs.dagster.io/guides/dagster/asset-versioning-and-caching](https://docs.dagster.io/guides/dagster/asset-versioning-and-caching)) now includes a section on user-provided data versions
+- The [Asset Versioning and Caching Guide](<[https://docs.dagster.io/guides/dagster/asset-versioning-and-caching](https://docs.dagster.io/guides/dagster/asset-versioning-and-caching)>) now includes a section on user-provided data versions
```

Comment:
Not familiar with `(<[https://some.link]>)` syntax-- normally I would just use `(https://some.link)`? But if it works np.

Reply:
didn't notice that, my editor added those. I guess bc it had the broken double link